testRearrangeCarGroup8.py

```diff
@@ -0,0 +1,94 @@
+import unittest
+import numpy as np
+from RearrangeCarGroup8Movements import RearrangeCars
+
+class TestRearrangeCars(unittest.TestCase):

```

Comment:
I would putit in __init__ but since the deadline is passed I think rearrange_all_cars() will do :D